### DISCUSSION TITLE
Add borderImage in theme

### DISCUSSION THREAD

**Discussion by gerborisova** [chakra-ui/chakra-ui#7851] [Q&A]
Hi I want to add borderImage (linear gradient) in a variant of a button by extending the theme and using defineStyleConfig. However I can't seem to find a way to use it, do you have any advice ?

**Comment by HaleyWardo** [comment]
Looks like it isn't supported by looking at this [comment](https://github.com/chakra-ui/chakra-ui/issues/2440#issuecomment-723946220). Although, this was 3 years ago, it may still be relevant.

**Reply by gerborisova** [reply]
As I understand you can use it directly in an element by using the sx prop... but I don't find anything about using it in extending the theme
